packages/roosterjs-editor-api/lib/utils/processList.ts

```diff
@@ -39,65 +34,61 @@ export default function processList(
                     editor.getElementAtCursor('LI', currentRange.endContainer) == parentLINode)
             ) {
                 relativeSelectionPath = getSelectionPath(parentLINode, currentRange);
-                // Chrome has some bad behavior when outdenting
-                // in order to work around this, we need to take steps to deep clone the current node
-                // after the outdent, we'll replace the new LI with the cloned content.
+                if (parentLINode.textContent === '') {
+                    // If the node is empty, we need to handle this special case.
+                    // Chromium will try to replace all empty spans with font tags
+                    // We should preserve where our cursor is so that in this case, we can keep the span around.
+                    const cursorNode = editor.getElementAtCursor();
+                    clonedCursorNode = cursorNode.cloneNode(true);
+                    cursorSelectionPath = getSelectionPath(cursorNode, currentRange);
+                }
                 clonedNode = parentLINode.cloneNode(true);
             }
         }
-
-        workaroundForChrome(editor);
     }
 
     let existingList = editor.getElementAtCursor('OL,UL');
     editor.getDocument().execCommand(command, false, null);
-    let newParentNode: Node;
-    editor.queryElements('.' + TEMP_NODE_CLASS, node => {
-        newParentNode = node.parentNode;
-        editor.deleteNode(node);
-    });
+    const newParentNode = editor.getElementAtCursor('LI');
     let newList = editor.getElementAtCursor('OL,UL');
     if (newList == existingList) {
         newList = null;
     }
 
-    if (newList && clonedNode && newParentNode) {
-        // if the clonedNode and the newLIParent share the same tag name
-        // we can 1:1 swap them
-        if (isHTMLElement(clonedNode)) {
-            if (
-                isHTMLElement(newParentNode) &&
-                clonedNode.tagName == (<HTMLElement>newParentNode).tagName
-            ) {
-                newList.replaceChild(clonedNode, newParentNode);
+    if (Browser.isChrome) {

```

Comment:
So this line will cause this block only runs for Chrome. It was running for all browers before. Is that ok?

Reply:
Technically, it would only run for chrome earlier, as `clonedNode` would never be set unless the browser was chromium.